src/DependencyInjection/Compiler/SearchQueryMappersPass.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace EzSystems\EzPlatformGraphQL\DependencyInjection\Compiler;
+
+use EzSystems\EzPlatformGraphQL\GraphQL\InputMapper\SearchQueryMapper;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+class SearchQueryMappersPass implements CompilerPassInterface
+{
+    const ID = 'EzSystems\EzPlatformGraphQL\GraphQL\InputMapper\SearchQueryMapper';
```

Comment:
I would probably remove the constant, and use `SearchQueryMapper::class` in the `if` below. Even more so since you use it right after ;)